usermanager/src/main/java/org/pesmypetcare/usermanager/datacontainers/pet/PetData.java

```diff
@@ -61,6 +67,7 @@ public void setGender(GenderType gender) {
 
     /**
      * The method that returns the pet breed.
+     *
```

Comment:
I see you have added this additional line... why?

Reply:
The IDE adds it automatically when I reformat the code. Either way when the javadoc is generated it doesn't matter if this line exists or not, so it doesn't hurt anybody being there. Also, I think having the description separated from the parameters by an empty line makes more easy to read the documentation inside the editor.